test/e2e/json-rpc/wallet_requestPermissions.spec.ts

```diff
@@ -1,58 +1,42 @@
-const { strict: assert } = require('assert');
-const {
-  defaultGanacheOptions,
-  withFixtures,
-  switchToNotificationWindow,
-  switchToOrOpenDapp,
-  unlockWallet,
-} = require('../helpers');
-const FixtureBuilder = require('../fixture-builder');
+import { strict as assert } from 'assert';
+import { withFixtures } from '../helpers';
+import FixtureBuilder from '../fixture-builder';
+import { loginWithBalanceValidation } from '../page-objects/flows/login.flow';
+import TestDapp from '../page-objects/pages/test-dapp';
 
 describe('wallet_requestPermissions', function () {
   it('executes a request permissions on eth_accounts event', async function () {
     await withFixtures(
       {
         dapp: true,
-        fixtures: new FixtureBuilder()
-          .withPermissionControllerConnectedToTestDapp()
```

Comment:
we should not use `withPermissionControllerConnectedToTestDapp` here because in this test we want to test the request to ask for dapp permission

Reply:
nice catch